amoro-ams/amoro-ams-server/src/main/resources/mysql/ams-mysql-init.sql

```diff
@@ -82,7 +82,7 @@ CREATE TABLE `table_identifier`
     `format`          VARCHAR(32)  NOT NULL COMMENT 'Table Format',
     PRIMARY KEY (`table_id`),
     UNIQUE KEY `table_name_index` (`catalog_name`,`db_name`,`table_name`)
-);
+) ROW_FORMAT=DYNAMIC;
```

Comment:
Maybe add engine and charset information for this table too:
```suggestion
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COMMENT 'Table identifier for AMS' ROW_FORMAT=DYNAMIC;
```